docs/Installing.asciidoc

```diff
@@ -302,19 +302,30 @@ zypper dup --from devel_openQA_Leap --allow-vendor-change
 
 
 === Installation
-You can install the main openQA server package using these commands.
+
+==== Preparations on SLE
+
+On SLE some repositories have to be added.
+You can either choose to add the development repositories <<_development_version_repository,as described above>>
+or the official repositories using SUSEConnect by running the following commands.
```

Comment:
I don't think it's a choice to use the development version repository *or* official repositories. A lot of dependencies are only in the SLE module and PackageHub so I don't think those are alternative. How did you come to that conclusion? Did you try it out?